easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'Rust'
+version = '1.52.1'
+
+homepage = 'https://www.rust-lang.org'
+description = """Rust is a systems programming language that runs blazingly fast, prevents segfaults,
+ and guarantees thread safety."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://static.rust-lang.org/dist/']
+sources = ['rustc-%(version)s-src.tar.gz']
+checksums = ['3a6f23a26d0e8f87abbfbf32c5cd7daa0c0b71d0986abefc56b9a5fbfbd0bf98']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
```

Comment:
`Python` and `pkg-config` are missing as a build dependency here

We don't have a `bare` Python, just using the regular one is fine here imho (we can't do that in more recent toolchains, where `Rust` has become a build dependency for `Python`...)

Reply:
Thanks for spotting that! 
I have seen the PR you opened, once the tests have passed I will merge it with this one here.

Reply:
Merged.